src/openrct2/rct12/RCT12.h

```diff
@@ -14,6 +14,7 @@
 #include "../common.h"
 #include "../object/Object.h"
 
+#include <src/openrct2/ride/RideTypes.h>
```

Comment:
`../ride/RideTypes.h`

Reply:
Argh, my IDE keeps auto-adding that :/

Reply:
I'll fix it never mind.

Reply:
Sorry, read that too late - I already pushed a fix for that and for formatting.

Reply:
never mind you got there first.